### PR TITLE
Refactor: updated prisma-cli references

### PR DESCRIPTION
**Pull request by ruheni**

### CODE REVIEW

**Comment by nikolasburk** [comment]
Thank you Alex! 🙏

File: content/200-concepts/050-overview/500-under-the-hood.mdx [hunk 1]
```diff
@@ -56,7 +56,7 @@ As an example, Prisma Client connects to the [query engine](../components/prisma
 
 ### Using custom engine binaries
 
-By default, all binaries are automatically downloaded into the `node_modules/@prisma/cli` folder when you install or update `@prisma/cli`. You might want to use a [custom binary](https://github.com/prisma/prisma-engines) file if:
+By default, all binaries are automatically downloaded into the `node_modules/@prisma` folder when you install or update `prisma`. You might want to use a [custom binary](https://github.com/prisma/prisma-engines) file if:
```
Comment (janpio) [inline content/200-concepts/050-overview/500-under-the-hood.mdx:59]: This still has an `@` too much.

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 2]
```diff
@@ -104,7 +104,7 @@ prisma version
 <CmdResult>
 
 ```
-Environment variables loaded from ./prisma/.env @prisma/cli : 2.13.0-beta.3 Current platform
+Environment variables loaded from ./prisma/.env prisma : 2.13.0-beta.3 Current platform
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx:107]: Is this missing line breaks intentionally?

File: content/200-concepts/050-overview/500-under-the-hood.mdx (hunk 1 above)
Comment (nikolasburk) [reply content/200-concepts/050-overview/500-under-the-hood.mdx:59]: I fixed this via this PR: https://github.com/prisma/docs/pull/1260

Note that the file paths here also still have an `@`, I'd assume but wasn't sure if they needed to be removed as well:

image

Maybe @ruheni you can validate and potentially fix the files paths if needed?

Comment (ruheni) [reply content/200-concepts/050-overview/500-under-the-hood.mdx:59]: The paths don't need to be fixed. The `@` is still part of the path to the packages in the `node_modules`
```bash
❯ npx prisma -v
Environment variables loaded from .env
prisma               : 2.16.0
@prisma/client       : 2.16.0
Current platform     : windows
Query Engine         : query-engine 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine     : migration-engine-cli 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine : introspection-core 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary        : prisma-fmt 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)Studio               : 0.346.0
```

Comment (janpio) [reply content/200-concepts/050-overview/500-under-the-hood.mdx:59]: Yep, this is coming from the `@prisma/engines` package.

**Pull request merged**